source/docs/prismic/routable-types.md

```diff
@@ -60,7 +61,45 @@ export default {
 +};
 ```
 
-Note we added a path field resolver that appends the shop's baseUrl  to the `url` field of our custom Prismic type.
+Note we added a path field resolver that appends the shop's baseUrl to the `url` field of our custom Prismic type.
+
+## Add GraphQL type to the dispatcher query
+
+Override `DispatcherQuery` if you havent already done so. You can find it under `src/web/theme/modules/Router/DispatcherQuery.gql`.
+
+Add a segment to `DispatcherQuery` that represents your newly created GraphQL type as noted below:
+
+```diff
+...
+query MatchUrls($url: String!, $params: QueryInput) {
+  route(url: $url) {
+    path
+    __typename
+    ... on RedirectEntity {
+      redirectTo
+      redirectType
+    }
+    ... on Product {
+      ...ProductFragment
+      ...ProductCacheControlFragment
+    }
+    ... on Category {
+      ...CategoryFragment
+      ...CategoryCacheControlFragment
+    }
+    ... on CmsPage {
+      ...CmsPageFragment
+      ...CmsPageCacheControlFragment
+    }
++    ... on Album {
++      ...AlbumFragment
++      ...AlbumCacheControlFragment # optional check note below for link
```

Comment:
I think the content of those fragment needs to be details (especially the
AlbumFragment)

Reply:
the `AlbumFragment` sure but the `AlbumCacheControlFragment` i have a link to the documentation which should be enough